packages/api/portfolioDrafts/submit/submitPortfolioDraft.test.ts

```diff
@@ -0,0 +1,69 @@
+import { GetCommand, DynamoDBDocumentClient, UpdateCommand } from "@aws-sdk/lib-dynamodb";
+import { APIGatewayProxyEvent } from "aws-lambda";
+import { mockClient } from "aws-sdk-client-mock";
+import { handler } from "./submitPortfolioDraft";
+import {
+  NO_SUCH_PORTFOLIO_DRAFT,
+  REQUEST_BODY_NOT_EMPTY,
+  PORTFOLIO_ALREADY_SUBMITTED,
+  PATH_PARAMETER_REQUIRED_BUT_MISSING,
+  DATABASE_ERROR,
+} from "../../utils/errors";
+
+const ddbMock = mockClient(DynamoDBDocumentClient);
+beforeEach(() => {
+  ddbMock.reset();
+});
+
+const validRequest: APIGatewayProxyEvent = {
+  pathParameters: { portfolioDraftId: "1234" },
+} as any;
+
+describe("Validation tests", () => {
+  it("should require path param", async () => {
+    const emptyRequest: APIGatewayProxyEvent = {} as any;
+    expect(await handler(emptyRequest)).toEqual(PATH_PARAMETER_REQUIRED_BUT_MISSING);
+  });
+
+  it("should return generic Error if exception caught", async () => {
+    ddbMock.on(GetCommand).rejects("Some error occurred");
+    expect(await handler(validRequest)).toEqual(DATABASE_ERROR);
+  });
+
+  it("should return REQUEST_BODY_NOT_EMPTY when request body is not empty", async () => {
+    const requestBodyShouldBeEmpty = {
+      hi: "This should be empty",
+    };
+    const request: APIGatewayProxyEvent = {
+      body: JSON.stringify(requestBodyShouldBeEmpty),
+      pathParameters: { portfolioDraftId: "1234" },
+    } as any;
```

Comment:
```suggestion
    const request: APIGatewayProxyEvent = {
      body: JSON.stringify(requestBodyShouldBeEmpty),
      ...validRequest,
    } as any;
```

?

Reply:
This change results in a typescript error 
```
'body' is specified more than once, so this usage will be overwritten.ts(2783)
submitPortfolioDraft.test.ts(39, 7): This spread always overwrites this property.
```

Reply:
I am confused when/where `validRequest` gets a body attribute but that's fine. Just flip them.

```
{
  ...validRequest,
  body: JSON.stringify(requestBodyShouldBeEmpty)
}
```